src/index.html

```diff
@@ -2,12 +2,359 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0">
+    <title>Bang & Olufsen</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap"
+      rel="stylesheet">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
+      rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header
+      class="page__header header"
+      id="header">
+      <div class="header__nav nav">
+        <img
+          src="images/Logo.svg"
+          alt="Logo"
+          class="nav__logo">
+
+        <div class="nav__buttons">
+          <a href="tel:+1 234 555-55-55"
+            class="phone nav__buttons--link">
+            <span class="phone__label">+1 234 555-55-55</span>
+            <img
+              src="images/Icon-Phone-call.svg"
+              alt="Icon-Phone"
+              class="nav__buttons--icon">
+          </a>
+          <a href="#menu"
+            class="burger nav__buttons--link">
+            <span class="burger__label">Open menu</span>
+          </a>
+        </div>
+      </div>
+      <h1 class="title header__title">
+        Your senses.<br>
+        Set free.
+      </h1>
+    </header>
+
+    <div
+      class="menu"
+      id="menu"
+    >
+      <div class="menu__header">
+        <img
+          src="images/Logo.svg"
```

Comment:
should be inside <header> tag

Reply:
Sorry doesn't really get it
Could You please explain